### PR TITLE
Add about page WITH TYPING ANIMATION

### PR DESCRIPTION
**Pull request by JulianaTennett**
Added about page with typed.js animation and looping facts

Added to home and nav

### CODE REVIEW

File: _scripts/main.js [hunk 1]
```diff
@@ -1,7 +1,47 @@
 import Nav from './modules/nav';
 import Homepage from './modules/homepage';
+import Typed from 'typed.js';
+// import Homepage from './modules/about';
```
Comment (beahuang) [inline _scripts/main.js]: when you import you need to call it something different import About from './modules/about

File: _layouts/base.html [hunk 2]
```diff
@@ -12,11 +12,15 @@
...
   <link href="https://fonts.googleapis.com/css?family=Asap:400,600" rel="stylesheet">
+  <!-- this is typer -->
+  <link rel="stylesheet" href="https://unpkg.com/typer-js/typer.css">
 </head>
 
 <body>
   {{ content }}
 
+  
+  <script src="https://unpkg.com/typer-js"></script>
```
Comment (beahuang) [inline _layouts/base.html]: you don't need this script if you are importing Typer from npm

File: about.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<div class="page--about">
+  {% include layout/header.html %}
+
+  <div class="content">
+
+    <div class="project-section">
+      <div class="about__container">
+
+        <!-- <div class="about__text--mask">
```
Comment (beahuang) [inline about.html]: what are all these comments

File: _layouts/base.html (hunk 2 above)
Comment (JulianaTennett) [reply _layouts/base.html]: leftover from typer.js before switch to typed.js

File: _scripts/main.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
 import Nav from './modules/nav';
 import Homepage from './modules/homepage';
+import About from './modules/about';
 
 $(document).ready(function() {
   new Nav();
   new Homepage();
-}); //this creates a jquery object on document then calling .ready() on that jquery object, which takes in a function tnats called afer the jquery obkect is loaded
+  new About();
+  // changeFact();
```
Comment (beahuang) [inline _scripts/main.js]: 😠

**Pull request merged**